app/views/notifications/_emergency_banner.html.erb

```diff
@@ -1,9 +1,9 @@
-<section id="emergency-banner-notification" class="<%= banner.campaign_class %>">
+<section id="emergency-banner-notification" class="<%= banner.campaign_class %>" role="banner">
```

Comment:
If we remove it, what do you think of bringing the section back? Or is there another role we could use instead? Maybe `region`?